ModelDerivative_01/instructions/task-1.md

```diff
@@ -15,11 +15,11 @@ To set the environment variables:
 
 1. Click the **Environment quick look** icon on the upper right corner of Postman.
 
-   ![Environment quick look icon](../images/task1-environment_quick_look_icon.png "Environment quick look icon")
+   ![Environment quick look icon](https://github.com/sujeet14788/forge-tutorial-postman/blob/Test-tutorial/ModelDerivative_01/images/task1-environment_quick_look_icon_test.png "Environment quick look icon")
```

Comment:
Pls use relative path.

Reply:
Path is showing correct in file.
Request you to check once again.